### ISSUE TITLE
Navigation bar not showing

### ISSUE THREAD

**Issue by meticajp** [gabrieltempass/streamlit-navigation-bar#28]
### Checklist

- [X] I have read the documentation at the [Wiki](https://github.com/gabrieltempass/streamlit-navigation-bar/wiki/API-reference).

### Description

Hi, I have been trying out the navigation bar and I've copied and pasted the code from git. I even downloaded the raw file but the nav bar is still not showing. 
![image](https://github.com/user-attachments/assets/11fb4921-0dd6-41b7-aad4-64094b4496f2)
![image](https://github.com/user-attachments/assets/e2284c51-75bb-4bab-87c8-11f77c601ba0)

As you can see here, the background color of the nav bar is shown but not the links.

**Comment by ArthurTech123** [comment]
It looks cause by streamlit 1.39.0
this issue has already reported in #27

**Comment by beckmarc** [comment]
Issue still exists with streamlit 1.41.1

**Comment by flight505** [comment]
Yes, spent an hour on the issue, came to this issues page and .. Issue still exists with streamlit 1.41.1 I can confirm it.. To bad...
Edit - so stupid, first I had to fix the # base.css in the package, after spending time on that i found that this package is no longer supported and a working fork has been continued on https://github.com/streamlit-community/streamlit-navigation-bar SHOULD ON THE FRONT OF THE README.md

**Comment by Jeremy-Qian** [comment]
I've the same problem the nav bar not showing! @meticajp

**Comment by lamnd09** [comment]
Using: streamlit-community-navigation-bar
solved the problem!

pip install streamlit-community-navigation-bar

**Comment by arithmic-vanshaj** [comment]
this worked for me too.
